SimplenoteUITests/NoteEditor.swift

```diff
@@ -89,7 +89,6 @@ class NoteEditor {
 
     class func undo() {
         app.textViews.element.tap(withNumberOfTaps: 2, numberOfTouches: 3)
-        app.otherElements["UIUndoInteractiveHUD"].children(matching: .other).element(boundBy: 1).children(matching: .other).element(boundBy: 1).tap()
```

Comment:
Double tap with three fingers now does the undo, no need to additionally tap `Undo` in the context menu.